src/plugins/rpd/plugin.json

```diff
@@ -0,0 +1 @@
+{ "defaultEnable": true }
```

Comment:
As far as I understand, this plugin saves files with .json extension? This may conflict with the existing JSON export, so we can't enable this plugin by default.

Reply:
fixed